tools/microtvm/vm/kitware-archive-latest.asc

```diff
@@ -0,0 +1,81 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+-----BEGIN PGP PUBLIC KEY BLOCK-----
+
```

Comment:
can we download the public key somewhere instead of putting in in?

Reply:
I did what i'd done previously on `docker/install/ubuntu_install_zephyr.sh`, wouldn't downloading the key defeat the purpose of it?

Reply:
I believe the goal here is to build the vmbox, like in the case of LLVM and other places, we download the pubkey from the source of truth, in this case from https://apt.kitware.com/ 
so that we don't have to maintain the pubkey in our own repo